assets/blocks/contact-teacher-block/contact-teacher.scss

```diff
@@ -5,22 +5,22 @@
 		display: block;
 		font-size: 24px;
 		font-weight: 700;
-		margin-bottom: 60px;
+		margin-bottom: 30px;
```

Comment:
Just out of curiosity, aren't we worried that changes like this one might break the layout of existing sites that use the block in post content?

Reply:
Right, just checked that one and added a small `box-sizing `tweak so the spacing is consistent. (This style is for the modal, which is the same for LM and the normal block.)